substrate/frame/session/benchmarking/src/mock.rs

```diff
@@ -131,6 +131,7 @@ impl onchain::Config for OnChainSeqPhragmen {
 
 #[derive_impl(pallet_staking::config_preludes::TestDefaultConfig)]
 impl pallet_staking::Config for Test {
+	type OldCurrency = Balances;
```

Comment:
Why don't you add this to `TestDefaultConfig` and be done with it? And later when you remove it nothing has to change.

Reply:
Can it have a default? Since its a runtime pallet (depends on system config)?